src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -26,6 +26,14 @@ public interface JsonApiMetricsConfig {
   @WithDefault("sort.type")
   String sortType();
 
+  @NotBlank
+  @WithDefault("serialization.json")
+  String serializationJson();
```

Comment:
I understand there is need for one name configuration, but why 2?

Reply:
One is for the metrics name and one is for the tag that indicates its serialization or deserialization. I've renamed them to make it clear.